languages/ruby/exercises/concept/exceptions/simple_calculator.rb

```diff
@@ -0,0 +1,12 @@
+class SimpleCalculator
+  ALLOWED_OPERATIONS = ['+', '-', '/', '*']
+
+  # TODO: Show how to define custom exceptions
+  # TODO: Explain why custom errors are subclasses of StandardError
+  # TODO: Explain why is not a good idea to rescue from Exception
+  UnsupportedOperation = Class.new(StandardError)
```

Comment:
Is this something we want to stub for the student? I would assume part of the exercise is to define a custom error type, or is this not true?

Reply:
Copy and paste from the example 😄

Reply:
😄